gitops/teamcity/README.md

```diff
@@ -0,0 +1,92 @@
+# TeamCity
+
+TeamCity is a build management and continuous integration server developed by [JetBrains](https://www.jetbrains.com/teamcity/). TeamCity is available in the [Azure Marketplace](https://azuremarketplace.microsoft.com/en-en/marketplace/apps/jetbrains.teamcity?tab=Overview). Follow the instructions below to deploy an instance of TeamCity on your Azure cloud to run continuous integration builds.
```

Comment:
Not sure if this included anywhere, but it might also be good to create a diagram showing the pipeline that it is being modeled after (e.g. App Code -> Team City -> ACR).

Reply:
@yradsmikham  Actually the guide is for setting the main GitOps pipeline, the one with manifest generation. I will add a diagram.